spinnaker_components/ensemble/ensemble_update.c

```diff
@@ -57,43 +55,36 @@ void ensemble_update(uint ticks, uint arg1) {
     }
 
     v_voltage = neuron_voltage(n);
-    v_delta = ( i_membrane - v_voltage ) * g_ensemble.dt_over_t_rc;
-    /* io_printf( IO_STD, "n = %d, J = %k, V = %k, dV = %k\n",
-                  n, i_membrane, v_voltage, v_delta );
-    */
-
-    v_voltage += v_delta;
+    v_delta = (i_membrane - v_voltage) * g_ensemble.exp_dt_over_t_rc;
 
     // Voltages can't go below 0.0
-    if( v_voltage < 0.0k ) {
+    v_voltage += v_delta;
+    if(v_voltage < 0.0k)
+    {
       v_voltage = 0.0k;
```

Comment:
Ah, I see...  yup, that now makes sense to me.  (and it's exactly the same as it used to be)